pkg/operator2/starter.go

```diff
@@ -82,13 +80,13 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 
 	v1helpers.EnsureOperatorConfigExists(
 		dynamicClient,
-		[]byte(osinResource),
-		osinv1alpha1.GroupVersion.WithResource("osins"),
+		[]byte(authConfigResource),
+		authv1alpha1.GroupVersion.WithResource("osins"),
 	)
 
-	operator := NewOsinOperator(
-		osinInformersNamespaced.Osin().V1alpha1().Osins(),
-		osinClient.OsinV1alpha1(),
+	operator := NewAuthenticationOperator(
+		authConfigInformers.Authentication().V1alpha1().AuthenticationOperatorConfigs(),
+		authConfigClient.AuthenticationV1alpha1(), // TODO: Why does update-codegen.sh still generate Osin stuff here?
```

Comment:
Where?

Reply:
Ah, I managed to resolve that but left the note there, my bad.